lib/fathead/mdn_css/parse.pl

```diff
@@ -158,7 +155,29 @@
             $description = $p->all_text;
             my $next = $p->next;
             if ( $next && $next->tag eq 'ul' ) {
-                $description .= $next->find('li')->map('all_text')->join(', ');
+                my $li = $next->at('li');
+                my $a = $li->at('a') if $li;
+                if ( $a && $a->text =~ /Initial value/ ) {
+                    $initial_value = $li->text;
+                    $initial_value =~ s/://;
+                    $initial_value = _build_initial_value($initial_value);
+                }
+                else {
+                    $description .=
+                      $next->find('li')->map('all_text')->join(', ');
+                }
+            }
+        }
+    }
+    else {
```

Comment:
@hchienjo it looks like there is now unescaped unicode in one of the articles (i.e. "content") -- Can you add a regex to replace `\\00` with `\\\\00`?

You'll see the output contains `\00A0` and this creates problems when the files is read by our DB. It needs to be `\\00A0`